documentation/docs/verifiable_credentials/verifiable_presentations.mdx

```diff
@@ -10,17 +10,157 @@ keywords:
+- **Method Type**: check the signing verification method has a particular type.
+- **Challenge**: check the challenge field matches this string.
+- **Domain**: check the domain field matches this string.
+- **Proof Purpose**: require a specific purpose on the proof. See [Proof Options](#proof-options).
+- **Allow Expired**: accept proofs where the current datetime is after their expiration. Default is to reject expired proofs.
+
+See [Proof Options](#proof-options) for more information on setting these properties as a holder when signing a verifiable presentation.
+
+### Subject-Holder Relationship {#subject-holder-relationship}
+
+Specifies the expected relationship between the holder that signed the verifiable presentation and the subject specified in each verifiable credential.
+
+The following options are available:
+
+- **`AlwaysSubject` (default)**: the holder DID that signed the presentation must match the [`credentialSubject` `id`](https://www.w3.org/TR/vc-data-model/#credential-subject) field in each of the attached credentials. This is the safest option which ensures holders may only present credentials that were directly issued to their DID. An error is thrown on a mismatch or if no subject `id` is present.
+- **`SubjectOnNonTransferable`**: the holder DID must match the subject only for credentials where the [`nonTransferable`](https://www.w3.org/TR/vc-data-model/#nontransferable-property) property is `true`. This is appropriate for accepting [bearer credentials](https://www.w3.org/TR/vc-data-model/#bearer-credentials) while still adhering to the specification.
+- **`Any`**: the holder DID is not required to have any kind of relationship to any credential subject. This option performs no checks and ignores the [`nonTransferable`](https://www.w3.org/TR/vc-data-model/#nontransferable-property) property.
+
+:::note
+
+See the [Verifiable Credentials Data Model Specification](https://www.w3.org/TR/vc-data-model/#subject-holder-relationships) for further discussion on the different subject-holder relationships.
+
+:::
+
+### Credential Validation Options {#credential-validation-options}
+
+These options specify conditions that all credentials in a verifiable presentation must satisfy.
+
+- **Expiry Date**: check that the [`expirationDate`](https://www.w3.org/TR/vc-data-model/#expiration) property, if present, is not before a specific datetime. Defaults to the current datetime if unset.
+- **Issuance Date**: check that that [`issuanceDate`](https://www.w3.org/TR/vc-data-model/#issuance-date) property, if present, is not after a specific datetime. Defaults to the current datetime if unset.
+- **Verifier Options**: see [Proof Verifier Options](#proof-verifier-options) for details.
+
+## Security Considerations {#security-considerations}
+
+### Replay Attacks {#replay-attacks}
```

Comment:
Excellent section. This gives a lot of the reasons why the abovementioned options are so important. Perhaps we should move this to the top in order to prime readers on the context much more.